### PR TITLE
Katas UI review - Preparing states

### PR DESCRIPTION
**Pull request by SoniaLopezBravo**

### CODE REVIEW

File: katas/content/preparing_states/parity_bitstrings/solution.md [hunk 1]
```diff
@@ -1,55 +1,55 @@
-There are multiple ways to approach this problem. In our first solution, we are going to use a recursive approach that was hinted at in the task.
+There are multiple ways to approach this problem. In your first solution, you're going to use a recursive approach that was hinted at in the task.
```
Comment (tcNickolas) [inline katas/content/preparing_states/parity_bitstrings/solution.md:1]: Please rewrite to emphasize that they're reading about these solutions, not that we expect them to write all of them

File: katas/content/preparing_states/three_states_two_qubits/solution.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
-* If the result is $\ket{0}$, the system collapses to a state that is a linear combination of basis states which had the extra qubit in state $\ket{0}$, i.e., the two qubits end up in the target state $\frac{1}{\sqrt3}\big(\ket{00} + \ket{01} + \ket{10}\big)$. 
-* If the result is $\ket{1}$, the system collapses to a state $\ket{11}$, so our goal is not achieved. The good thing is, this only happens in 25% of the cases, and we can just reset our qubits to the $\ket{00}$ state and try again.
+Finally, you measure just the extra qubit; this causes a partial collapse of the system to the state defined by the measurement result:
+* If the result is $\ket{0}$, the system collapses to a state that is a linear combination of basis states which had the extra qubit in state $\ket{0}$, that is, the two qubits end up in the target state $\frac{1}{\sqrt3}\big(\ket{00} + \ket{01} + \ket{10}\big)$. 
+* If the result is $\ket{1}$, the system collapses to a state $\ket{11}$, so your goal isn't achieved. The good thing is, this only happens in $25%$ of the cases, and you can just reset your qubits to the $\ket{00}$ state and try again.
```
Comment (tcNickolas) [inline katas/content/preparing_states/three_states_two_qubits/solution.md:20]: Escape %

File: katas/content/preparing_states/unequal_superposition/solution.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+If you were to apply the $R_x$ gate to a qubit in the $\ket{0}$ state, you'd introduce complex coefficients to the amplitudes, which is clearly not what you're looking for. Similarly, the $R_z$ gate introduces only a global phase when applied to $\ket{0}$ state, so you can rule it out as well. This leaves only the $R_y$ as a starting point for the solution.
 
-Applying the $R_y$ gate to the $\ket{0}$ state, we get:
+Applying the $R_y$ gate to the $\ket{0}$ state, you get:
 $$R_y(\theta) \ket{0} = 
 \begin{bmatrix} \cos\frac{\theta}{2} & -\sin\frac{\theta}{2} \\ \sin\frac{\theta}{2} & \cos\frac{\theta}{2} \end{bmatrix} \begin{bmatrix} 1 \\ 0 \end{bmatrix} = 
 \begin{bmatrix} \cos\frac{\theta}{2} \\ \sin\frac{\theta}{2} \end{bmatrix} = \cos\frac{\theta}{2}\ket{0} + \sin\frac{\theta}{2}\ket{1}$$
 
-Therefore, applying the $R_y(2\alpha)$ gate to $\ket{0}$ is the solution to our problem. 
+Therefore, applying the $R_y(2\alpha)$ gate to $\ket{0}$ is the solution to your problem.
```
Comment (tcNickolas) [inline katas/content/preparing_states/unequal_superposition/solution.md:15]:
```suggestion
Therefore, applying the $R_y(2\alpha)$ gate to $\ket{0}$ is the solution to this problem. 
```

File: katas/content/preparing_states/wstate_arbitrary/solution.md [hunk 4]
```diff
@@ -48,31 +48,31 @@ But we would express the rest of the controlled rotations as the operation that
...
+As you can see, if the extra qubit is in the $\ket{0}$ state, the main 3 qubits that you're concerned about are in the right $\ket{W_3}$ state. 
 
-What happens if we measure just the extra qubit? This causes a partial collapse of the system to the state defined by the measurement result:
-* If the result is $\ket{0}$, the system collapses to the $\ket{W_3}$ state - which is exactly what we wanted to achieve.
-* If the result is $\ket{1}$, the system collapses to a state $\ket{000}$, so our goal is not achieved. The good thing is, this only happens in 25% of the cases, and we can just try again.
+What happens if you measure just the extra qubit? This causes a partial collapse of the system to the state defined by the measurement result:
+* If the result is $\ket{0}$, the system collapses to the $\ket{W_3}$ state - which is exactly what you wanted to achieve.
+* If the result is $\ket{1}$, the system collapses to a state $\ket{000}$, so your goal is not achieved. The good thing is, this only happens in $25%$ of the cases, and you can just try again.
```
Comment (tcNickolas) [inline katas/content/preparing_states/wstate_arbitrary/solution.md:67]: Escape

**Review by tcNickolas — CHANGES_REQUESTED** [review]
This PR seems to have more issues than the previous ones relatively to the number of files. Please make sure that you're previewing the files to check that the edits don't introduce formatting issues, like the missing escape for % inside formulas, and that no new typos are introduced, like the missing spaces between "you" and the next word.

**Comment by SoniaLopezBravo** [comment]
> This PR seems to have more issues than the previous ones relatively to the number of files. Please make sure that you're previewing the files to check that the edits don't introduce formatting issues, like the missing escape for % inside formulas, and that no new typos are introduced, like the missing spaces between "you" and the next word.

@tcNickolas I appreciate the honesty in your comment 👍🏻 I got overconfident and reviewed this kata too fast, I'll be more careful next time.

File: katas/content/preparing_states/wstate_arbitrary/solution.md [hunk 5]
```diff
@@ -48,31 +48,31 @@ But we would express the rest of the controlled rotations as the operation that
 
 ### Post-selection Solution
 
-Let's assume that we know how to prepare the $W$ state for $N = 2^k$ (we've discussed this in the previous task), and figure out how to use this knowledge as a building block for solving this task.
+Let's assume that you know how to prepare the $W$ state for $N = 2^k$ (you've discussed this in the previous task), and figure out how to use this knowledge as a building block for solving this task.
```
Comment (tcNickolas) [inline katas/content/preparing_states/wstate_arbitrary/solution.md:51]:
```suggestion
Let's assume that you know how to prepare the $W$ state for $N = 2^k$ (see the previous task), and figure out how to use this knowledge as a building block for solving this task.
```

File: katas/content/preparing_states/wstate_arbitrary/solution.md [hunk 6]
```diff
@@ -48,31 +48,31 @@ But we would express the rest of the controlled rotations as the operation that
...
 
-Let's look at the smallest possible case for which $N \neq 2^k$: $N = 3$ (we'll be able to generalize our solution for this case to an arbitrary number of qubits). The target $W$ state looks like this:  
+Let's look at the smallest possible case for which $N \neq 2^k$: $N = 3$ (you'll be able to generalize the solution for this case to an arbitrary number of qubits). The target $W$ state looks like this:  
 
 $$\ket{W_3} = \frac{1}{3}\big(\ket{100} + \ket{010} + \ket{001}\big)$$
 
-We will start by finding the smallest power of 2 $P$ which is greater than or equal to $N$; for our case $N = 3$ this power will be $P = 4$. We will allocate an extra $P - N$ qubits and use the solution of the previous task to prepare the $W_P$ state that looks as follows (with the state of the extra qubit highlighted in bold):  
+You'll start by finding the smallest power of 2 $P$ which is greater than or equal to $N$; for this case $N = 3$ this power will be $P = 4$. You'll allocate an extra $P - N$ qubits and use the solution of the previous task to prepare the $W_P$ state that looks as follows (with the state of the extra qubit highlighted in bold):
```
Comment (tcNickolas) [inline katas/content/preparing_states/wstate_arbitrary/solution.md:57]:
```suggestion
You'll start by finding the smallest power of 2 $P$ which is greater than or equal to $N$; for this case $N = 3$, so this power will be $P = 4$. You'll allocate an extra $P - N$ qubits and use the solution of the previous task to prepare the $W_P$ state that looks as follows (with the state of the extra qubit highlighted in bold):  
```

File: katas/content/qec_shor/index.md [hunk 7]
```diff
@@ -335,14 +335,14 @@ To correct a $Z$ error, you can no longer simply apply a $Z$ gate to the affecte
...
 > You can represent the state with this error as some superposition of four orthogonal states: $\ket{\psi_L}$, $X\ket{\psi_L}$, $Z\ket{\psi_L}$, and $XZ\ket{\psi_L}$ (this is possible because the Pauli matrices $I, X, Y, Z$ form a basis for $2 \times 2$ Hermitian matrices, so the operator representing the error can be decomposed as their linear combination).
 > 
 > Doing the set of parity measurements to detect $X$ and $Z$ errors will collapse the state $E\ket{\psi_L}$ into other of these four states, yielding measurement results that match that state. For example, if parity measurements yield a set of results indicating that an $X$ error happened, the state $E\ket{\psi_L}$ will collapse to $X\ket{\psi_L}$. 
 >
-> Then, you can use the results of parity measurements to correct the error that they indicate, bringing our state back to the error-free state $\ket{\psi_L}$.
+> Then, you can use the results of parity measurements to correct the error that they indicate, bringing your state back to the error-free state $\ket{\psi_L}$.
```
Comment (tcNickolas) [inline katas/content/qec_shor/index.md:345]:
```suggestion
> Then, you can use the results of parity measurements to correct the error that they indicate, bringing the state back to the error-free state $\ket{\psi_L}$.
```

**Review by tcNickolas — APPROVED** [review]
Looks good overall, just a couple of suggestions.

Thank you!

**Review by swernli — APPROVED** [review]

**Pull request merged**